Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
-      # set log level temporarily for npm's `make install`
-      ENV["NPM_CONFIG_LOGLEVEL"] = "verbose"
-      # unset prefix temporarily for npm's `make install`
-      ENV.delete "NPM_CONFIG_PREFIX"
-
-      cd buildpath/"npm_install" do
-        system "./configure", "--prefix=#{libexec}/npm"
```

Comment:
@DomT4: The [npm@5.2 branch](https://github.com/Homebrew/homebrew-core/compare/master...chrmoritz:npm@5.2) is already rebased on that, but I thought we would track upstream node8 releases, so that this will ship likely next week together with the node8 update including npm@5.2.